src/components/SelectionList/Search/ActionCell.tsx

```diff
@@ -7,27 +7,42 @@ import useLocalize from '@hooks/useLocalize';
 import useStyleUtils from '@hooks/useStyleUtils';
 import useTheme from '@hooks/useTheme';
 import useThemeStyles from '@hooks/useThemeStyles';
+import * as SearchUtils from '@libs/SearchUtils';
 import variables from '@styles/variables';
 import CONST from '@src/CONST';
+import type {TranslationPaths} from '@src/languages/types';
+import type {SearchTransactionAction} from '@src/types/onyx/SearchResults';
+
+const actionTranslationsMap: Record<SearchTransactionAction, TranslationPaths> = {
+    view: 'common.view',
+    review: 'common.review',
+    done: 'common.done',
+    paid: 'iou.settledExpensify',
+    hold: 'iou.hold',
+    unhold: 'iou.unhold',
```

Comment:
I'm sorry, but Im not sure I understand what is the best followup now.
Should I add new translation keys for Search for these 2 cases and set them to what you suggest.
Or should I modify these 2 existing keys for Spanish? sorry my knowledge of Spanish is 1/100 😅

Reply:
Let's create new ones since we also need this adjective version for the headers. Although, I do see that we use it in the [context menu action](https://github.com/Expensify/App/blob/4eb569959485e23a6f4f717c60b8285aa7785811/src/pages/home/report/ContextMenu/ContextMenuActions.tsx#L277) too and I think is incorrect.